src/providers/etherscan.js

```diff
@@ -1,29 +1,95 @@
+import rp from 'request-promise';
+import logger from '../logger';
+import { getAccessToken } from '../config';
+
+const path = require('path');
+
+const fs = (require('fs'));
+
+const expectedResponce = {
```

Comment:
`expectedResponse` not `expectedResponce`

Reply:
done